src/test/java/org/jqassistant/plugin/jee/cdi/test/set/beans/scope/javax/JavaxTypeWithDependentField.java

```diff
@@ -0,0 +1,12 @@
+package org.jqassistant.plugin.jee.cdi.test.set.beans.scope.javax;
+
+import javax.enterprise.context.Dependent;
+
+import org.jqassistant.plugin.jee.cdi.test.set.beans.scope.ProducedBean;
+
+public class JavaxTypeWithDependentField {
+
+    @Dependent
+    ProducedBean producedBean;
```

Comment:
Please verify if this is supported by CDI or if Produces is required in addition. If not then this class and according rules shall be removed.